server/app-engine/static/service-alert-ui.js

```diff
@@ -0,0 +1,613 @@
+
```

Comment:
This is too much new code for me to review in-depth in a reasonable amount of time.